mvp.feature

```diff
@@ -191,295 +290,453 @@ Feature: User with no Author identity in the Authority Register for Personas see
     And they see that the correct authors have the expected value for Corresponding Author
     And they see that the Other Contributors are listed alphabetically by Surname
     And they see that the Other Contributors have the correct Role, Name and Institution
-    And the Next button is enabled
 
-    # Corresponding Author is not in fact present in the Datacite data, but probably will be soon
+  @417
+  Scenario: User navigates to <Contributor> tab
```

Comment:
Is <Contributor> intentional? Not sure that this works as Contributor is not specified in an Examples table and is in the Scenario